apps/dashboard/src/fairness/utils.ts

```diff
@@ -64,13 +68,26 @@ export function generateRandomMetrics(
   const binSize = Math.max(...request.binVector);
   const bins: number[] = new Array(binSize + 1)
     .fill(0)
-    .map(() => Math.random());
-  // bins[2] = undefined;
+    .map(() => Math.random() / 3 + 0.33);
+  const binBounds: IBounds[] = bins.map((bin) => {
+    return {
+      lower: bin - Math.pow(Math.random() / 3, 2),
```

Comment:
This looks like it's in product code? Or am I just not familiar with how TS projects are laid out?

Reply:
This function was already generating random points for local testing and testing on Azure Blob. I just expanded it to also generate bounds